packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.Props.ts

```diff
@@ -0,0 +1,19 @@
+import { IVerticalDividerClassNames } from './VerticalDivider.classNames';
+
+export interface IDivider {
+
+}
+
+export interface IVerticalDividerProps {
+  /**
+   * Optional callback to access the IDivider interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IDivider) => void;
+
+  /**
+   * Optional classnames to apply on the elements of the divider. This can be used to apply
+   * custom styles to the divider.
+   */
+  classNames?: Partial<IVerticalDividerClassNames>;
```

Comment:
Shouldn't this be a function that takes in theme?

Reply:
Yeah that makes sense, it is the most flexible option. It is a bit annoying to import the default styles when you just want to change the height, but I think it's ok.

Where does that className come from?